satpy/scene.py

```diff
@@ -1023,6 +1031,52 @@ def show(self, dataset_id, overlay=None):
             img.show()
         return img
 
+    def to_geoviews(self, gvtype=gv.Image, datasets=None, kdims=None, vdims=None, dynamic=False):
```

Comment:
We can't have geoviews as a required dependency on satpy. At least not right now. So if you could move the new imports inside this function that would be preferred.

Reply:
Yes will do. I wasn't sure how you guys handled that.

Reply:
For conversion methods like this where additional dependencies are needed we tend to do in-method imports. Another alternative, if it ends up being cleaner, is to do the import at the top inside a try/except with a `HAS_GEOVIEWS = True` variable if imported, `False` if not imported, that gets checked inside the function using it.